components/api_version/src/api_v2.rs

```diff
@@ -235,6 +235,16 @@ impl ApiV2 {
         Ok(Key::split_on_ts_for(key)?)
     }
 
+    pub fn add_prefix(key: &[u8]) -> Vec<u8> {
+        let mut apiv2_key = Vec::with_capacity(ApiV2::get_encode_len(key.len() + 2));
+        apiv2_key.push(RAW_KEY_PREFIX);
+        let mut tmp_buf = [0; MAX_VARINT64_LENGTH];
+        let written = NumberCodec::encode_var_u64(&mut tmp_buf, DEFAULT_KEY_SPACE_ID);
```

Comment:
Suggest to try making written be constant. Or just write a 0 and verify it's always equal to var_u64 in unit test.

Reply:
Let's make key space fixed.

Reply:
done